changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+bump: patch
```

```suggestion
- bump: patch
```
This is required due to how this file's parsed by `yaml-changelog`

---

Question: I was going to write that in the commit title ("FIXES #issue") , but I looked at your [commit log](https://github.com/PolicyEngine/policyengine-api/commits/master/) and the convention seemed to actually be "thing I did (#issue)" (for instance [Fix error debugging (https://github.com/PolicyEngine/policyengine-api/pull/1943)](https://github.com/PolicyEngine/policyengine-api/commit/6ccbaf3e60b74857d224f6bc11d2af8f062efde0) which was why I wrote the commit message that way.

Is there some automation that replaces the title and/or am I misunderstanding what you are asking for there?

---

Right, I did see your comment in Slack on #1954. Weirdly, I tried responding last night, but my message doesn't appear as of today, though I see Nikhil ended up giving some context. If nothing becomes apparent, feel free to let me know. I can run locally and see if I can figure out what's going on.